framework/physics/physics_material/multi_group_xs/multi_group_xs.cc

```diff
@@ -92,32 +71,60 @@ MultiGroupXS::ExportToChiXSFile(const std::string& file_name, const double fissi
     Print1DXS(ofile, "SIGMA_F", scaled_sigma_f, 1.0e-20);
     if (NumPrecursors() > 0)
     {
+      // Compute prompt/delayed fission neutron yields
+      const auto& sigma_f = SigmaFission();
+      const auto& nu_prompt_sigma_f = NuPromptSigmaF();
+      const auto& nu_delayed_sigma_f = NuDelayedSigmaF();
+
+      std::vector<double> nu_prompt, nu_delayed;
+      for (size_t g = 0; g < NumGroups(); ++g)
+      {
+        nu_prompt.emplace_back(sigma_f[g] > 0.0 ? nu_prompt_sigma_f[g] / sigma_f[g] : 0.0);
+        nu_delayed.emplace_back(sigma_f[g] > 0.0 ? nu_delayed_sigma_f[g] / sigma_f[g] : 0.0);
+      }
+
+      // Get decay constants and fractional yields
+      std::vector<double> lambda, gamma;
+      for (const auto& precursor : Precursors())
+      {
+        lambda.emplace_back(precursor.decay_constant);
+        gamma.emplace_back(precursor.fractional_yield);
+      }
+
       Print1DXS(ofile, "NU_PROMPT", nu_prompt, 1.0e-20);
       Print1DXS(ofile, "NU_DELAYED", nu_delayed, 1.0e-20);
-      //      Print1DXS(ofile, "CHI_PROMPT", chi_prompt, 1.0e-20);
+      // Print1DXS(ofile, "CHI_PROMPT", chi_prompt, 1.0e-20);
```

Comment:
I personally recommend to delete lines like this. If is not used remove it. When it is needed have a proper change set doing the right thing. Lines like this like to rot over time.

Reply:
Fair point. It will likely come back at a later date, but we can always add it back, when needed.